server/core/store.go

```diff
@@ -33,6 +33,7 @@ const (
 	storePersistInterval = 5 * time.Minute
 	initialMinSpace      = 8 * units.GiB // 2^33=8GB
 	slowStoreThreshold   = 80
+	awakenStoreInterval  = 30 * time.Second
```

Comment:
why is  it  30s ?

Reply:
Its double to the default slow-score tick in TiKV side, where one full round for updating slow-score is (500ms * 30 == 15s). And also, according to testing results, it's proved to be an appropriate interval here.